packages/components/src/toggle-group-control/toggle-group-control/component.tsx

```diff
@@ -52,7 +52,7 @@ function UnconnectedToggleGroupControl(
 	const [ controlElement, setControlElement ] = useState< HTMLElement >();
 	const refs = useMergeRefs( [ setControlElement, forwardedRef ] );
 	const selectedRect = useTrackElementOffsetRect(
-		value ? selectedElement : undefined
+		value || value === 0 ? selectedElement : undefined
```

Comment:
This is a logical error in the component and not related to this PR. I'll open a separate PR for it tomorrow.